network-api/networkapi/wagtailpages/rss.py

```diff
@@ -0,0 +1,38 @@
+from django.conf import settings
+from django.contrib.syndication.views import Feed
+from django.utils.feedgenerator import Atom1Feed
+
+from .models import BlogPage
+
+
+class RSSFeed(Feed):
+    """
+    Blog page RSS feed, using the content:encoded serializer above
+    """
+
+    title = 'Mozilla Foundation Blog'
+    link = 'https://foundation.mozilla.org/blog/'
+    feed_url = 'https://foundation.mozilla.org/blog/rss/'
+    description = 'The Mozilla Foundation Blog'
+
+    def items(self):
+        return BlogPage.objects.live().public()[:settings.FEED_LIMIT]
+
+    def item_title(self, item):
+        return item.title
+
+    def item_link(self, item):
+        return item.full_url
+
+    def item_description(self, item):
+        return item.get_meta_description()
```

Comment:
If there's no meta description, I think we should return the first...140(?) characters of the content.

Reply:
yeaaaaaaaah that's not something that's easy to do given that streamfield yields HTML code, and there's no guarantee the first 140 characters are even text, rather than an `<img src="..." alt="...">`, which is why I went with meta description.

I can probably render out the body, run it through the python html library and then extract a starting snippet, but then we probably want to cache that somehow, so that we're not computing it constantly once we have a reasonable number of RSS readers hooked into this.